Signal/src/util/OWSWindowManager.m

```diff
@@ -67,9 +67,6 @@ @interface OWSWindowManager ()
 
 @property (nonatomic, nullable) UIViewController *callViewController;
 
-@property (nonatomic, nullable) UIResponder *rootWindowResponder;
```

Comment:
> since becomeFirstResponder is called on window.rootViewController

But we don't want it called on the window's `rootViewController`, we want it called on the "frontmost" vc, right?